src/docs/development/ui/animations/index.md

```diff
@@ -1,77 +1,110 @@
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/yI-8QHpGIP4?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-AnimatedContainer
+To create a custom implicit animation, watch
+[Creating your own custom implicit animations with TweenAnimationBuilder][].
+(Also published as a [_companion article_][article3].)
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/9hltevOHQBw?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-Opacity, including the implicit AnimatedOpacity widget
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/6KiPEqzJIKQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/pK738Pg9cxc?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-FadeInImage
+To create an explicit animation (where you control the animation,
+rather than letting the framework control it), perhaps
+you can use one of the built-in explicit animations classes.
+For more information, watch
+[Making your first directional animations with
+built-in explicit animations][].
+(Also published as a [_companion article_][article4].)
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/Be9UH1kXFDw?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-Hero
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/CunyH6unILQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/9z_YNlRlWfA?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-Transform
+If you need to build an explicit animation from scratch, watch
+[Creating custom explicit animations with
+AnimatedBuilder and AnimatedWidget][].
+(Also published as a [_companion article_][article5].)
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/N-RiyZlv8v8?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-AnimatedBuilder
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/fneC7t4R_B0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></center>
+
+For a deeper understanding of just how animations work in Flutter, watch
+[Animation deep dive][].
+(Also published as a [_companion article_][article6].)
```

Comment:
There's also the technical overview of the animations library here, that may be worth linking to: https://master-api.flutter.dev/flutter/animation/animation-library.html